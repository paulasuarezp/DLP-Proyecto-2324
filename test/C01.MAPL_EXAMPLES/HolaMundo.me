//*
Ejemplo HOLAMUNDO.
*//

class HOLAMUNDO;
	
	create
		main; 
			
	feature	main is
		/*local
		do
			print 'H', 'o', 'l', 'a', ' ', 'M', 'u', 'n', 'd', 'o', '!';
			println;
	end /*main
	
end /*HOLAMUNDO

run main(); 