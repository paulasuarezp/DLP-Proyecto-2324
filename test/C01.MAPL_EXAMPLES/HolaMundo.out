#source "HolaMundo.me"
#func main
#ret void
CALL main
HALT

#line 10
main:

#line 13
PUSHB 72	' 'H'
OUTB
PUSHB 111	' 'o'
OUTB
PUSHB 108	' 'l'
OUTB
PUSHB 97	' 'a'
OUTB
PUSHB 32	' ' '
OUTB
PUSHB 77	' 'M'
OUTB
PUSHB 117	' 'u'
OUTB
PUSHB 110	' 'n'
OUTB
PUSHB 100	' 'd'
OUTB
PUSHB 111	' 'o'
OUTB
PUSHB 33	' '!'
OUTB

#line 14
PUSHB 10
OUTB

#line 14
RET 0, 0, 0
